tests/integration/test_reshape.py

```diff
@@ -132,13 +146,91 @@ def test_1d(self):
             np.reshape(self.anp, (200,)),
         )
 
-    def test_ravel(self):
+    @pytest.mark.xfail
+    @pytest.mark.parametrize("order", ("C", "F", "A", "K", None), ids=str)
+    def test_ravel(self, order):
+        # In Numpy, pass with 'K'
+        # In cuNumeric, when order is 'K', raise ValueError:
+        # order 'K' is not permitted for reshaping
```

Comment:
is it only `mode="k"` that differs? Then the other modes should get a regular, passing test, and `mode="k"` should get a dedicated `xfail` test

Reply:
Fixed.